src/main/java/com/olgaboiar/mint/Router.java

```diff
@@ -3,43 +3,57 @@
 import com.olgaboiar.mint.handlers.*;
 
 import java.io.IOException;
-import java.util.HashMap;
-import java.util.Map;
-import java.util.Set;
+import java.util.*;
 
 public class Router {
     private Map<String, Map<String, IHandler>> routes = new HashMap<>();
+//    Map<String, IHandler> methodHandlers;
 
     public Router() {
         registerTestRoutes();
     }
 
     public Response route(Request request) throws IOException {
-        Map<String, IHandler> methodList = routes.get(request.getRequestedFile());
-        if (methodList == null) {
+        Map<String, IHandler> methodHandlers = routes.get(request.getUri());
+        if (methodHandlers == null) {
```

Comment:
Perhaps we can encapsulate "methodList == null" and "methodHandlers == null" into:

```java
if(methodExist?(methodList))...
```

Reply:
refactored in https://github.com/olgaboiar/MintHTTPServer/pull/9/commits/a8006851a1fedaf46447799896da0d08d709828f